cmd/postgres-operator/main.go

```diff
@@ -40,31 +41,66 @@ import (
 	"github.com/crunchydata/postgres-operator/pkg/apis/postgres-operator.crunchydata.com/v1beta1"
 )
 
-// assertNoError panics when err is not nil.
-func assertNoError(err error) {
+// must panics when err is not nil.
+func must(err error) { need(0, err) }
```

Comment:
So use `must` when we don't care about the value at all and just want to make sure the error is nil